src/Duo.Clients.Wpf/Presentation/ManutenzioneTrattamentoViewModel.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Duo.Domain.ViewModels.Trattamenti;
+using System;
+using System.Text;
+using Topics.Radical.ComponentModel.Messaging;
+using Topics.Radical.Windows.Presentation;
+using Topics.Radical.Windows.Presentation.Messaging;
+
+namespace Duo.Clients.Wpf.Presentation
+{
+    class ManutenzioneTrattamentoViewModel : AbstractViewModel
+    {
+        readonly IMessageBroker broker;
+
+        public TrattamentoView Trattamento
+        {
+            get { return this.GetPropertyValue(() => this.Trattamento); }
+            set { this.SetPropertyValue(() => this.Trattamento, value); }
+        }
+
+        public string WindowTitle
+        {
+            get { return this.GetPropertyValue(() => this.WindowTitle); }
+            set { this.SetPropertyValue(() => this.WindowTitle, value); }
+        }
+        public ManutenzioneTrattamentoViewModel(IMessageBroker broker)
+        {
+            this.broker = broker;
+            this.WindowTitle = (Trattamento.Id == Guid.Empty) ? "Inserimento Trattamento" : "Manutenzione Trattamento";
+        }
+
+        private string ControllaInserimento()
```

Comment:
Radical ha un motore di validazione: https://github.com/RadicalFx/Radical/wiki/Validation-and-Validation-Services